README.md

```diff
@@ -20,60 +20,70 @@
+from the releases page[^releases].  Save the former in a directory
+that is in already `PATH` so the extension works out of the box.
 
-In VSCode open the `extension` view and select the three dots on the top right of the sidebar.
+[^releases]: Available soon.
 
-Select `Install from VSIX ...` and select the `superbol-vscode-platform.vsix` file.
+### Adding the extension to VSCode
 
-### Build
+Open VSCode and open the extensions view.
 
-Get the `superbol` executable and install it in `/usr/local/bin`.
+In the sidebar, click on the three dots (`⋅⋅⋅`) on the top right-hand
+side (just above "`search`").
 
-Clone the source code of this extension:
-```bash
-git clone https://github.com/OCamlPro/superbol-vscode-platform.git
-```
+Select "`Install from VSIX…`" and pick
+`superbol-vscode-platform.vsix`.
 
-Go to the created folder and install the dependencies with
-```bash
-cd superbol-vscode-platform
-yarn install
-```
+### Configuring the extension
 
-Finally build the extension with:
-```bash
-make vsix-debug
-```
+If you have installed the `superbol-free` executable in a directory
+that is already in `PATH`, then you have nothing to do, the extension
+will work out of the box.
 
-### Add the extension to VSCode
+Otherwise get the path to the `superbol` executable and copy it.
 
-Open VSCode and go to the extension view.
+Go to your VSCode settings and in the extension submenu select
+`Superbol COBOL`.
 
-In the sidebar click on the three dots on the top right (just above `search`) and select
-`install from VSIX ...` and select the `superbol-vscode-platform.vsix` generated from
-this extension.
+In the `superbol` field, past the path to the `superbol-free`
+executable.
 
-### Configure the extension
+You can check the documentation on using the extension on [this
+page](https://ocamlpro.github.io/superbol-studio-oss/sphinx).
 
-If you have installed the `superbol` executable in `/usr/local/bin` then you have nothing to do,
-the extension will work out of the box.
+## GNU/Emacs mode
 
-Otherwise get the path to the `superbol` executable and copy it.
+You can check the documentation on using the Superbol LSP with
+GNU/Emacs on [this
+page](https://ocamlpro.github.io/superbol-studio-oss/sphinx/emacs).
 
-Go to your VSCode settings and in the extension submenu select `Superbol COBOL`.
+## Building from Sources
 
-In the `superbol` field past the path to the `superbol` executable.
+You first need to install a few external dependencies to build the LSP server and the VSCode extension from sources.
 
-You can check the documentation on using the extension on [this page](https://ocamlpro.github.io/superbol-vscode-platform/sphinx).
+1. First, you need to install and initialize [opam](https://opam.ocaml.org/);
 
-## GNU/Emacs mode
+1. Then you need a decent version of our build tool [drom](https://ocamlpro.github.io/drom/).  The easiest way to have it running is via the following command:
+
+   ```bash
+   opam pin add https://github.com/OCamlPro/drom.git
```

Comment:
Since `drom` seems to change its behavior regularly, we should maybe advertise a specific commit here?

Reply:
I thought about this, but considering its current development status (who develops it, number of users for projects of this size), I think we intend to always use the latest development version.

Reply:
When building an old branch (e.g. fixing a bug after release) the "latest development version" now may not be the "latest development version". I'm mentioning this because we have had issues with under-specified development versions of dependencies and bisecting with Alt-Ergo.